c_reference/include/conv1d.h

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#ifndef __CONV1D_H__
+#define __CONV1D_H__
+
+/**
+ * @brief Model parameters for the 1D Convolution Layer
+ * @var   W    pointer to convolutional weights W, size for regular = out_channels*in_channels*kernel_size, size for depth based = out_channels*kernel_size
+ * @var   B    pointer to the bias vector for the convolution, shape = [out_channels]
+ */
+typedef struct ConvLayers_Params {
+  float* W;
+  float* B;
+} ConvLayers_Params;
+
+/**
+ * @brief Model definition for the 1D Convolution Layer
+ * @param[out]   output_signal    pointer to the output signal, size = out_time * out_channels
+ * @param[in]    out_time            number of time steps in the output
```

Comment:
Please remove the extra spacing here and in other places as well. Let's keep these aligned.